frontend/src/app/menu/menu.component.ts

```diff
@@ -28,6 +28,10 @@ export class MenuComponent implements OnInit, OnDestroy {
     this.router.navigate(['']);
   }
 
+  searchButtonOnClick(): void {
+    this.router.navigate(['search']);
```

Comment:
shouldnt search feature be available on every page in our application? ofc for logged in user. having a separate page for it sounds wrong

Reply:
but we should have a page for search results